src/UI/pages/UnlockPage/index.tsx

```diff
@@ -1,14 +1,16 @@
 import React, { useEffect } from 'react';
 import classNames from 'classnames';
-import globalStyles from 'assets/sass/main.scss';
+
 import { useGetLoginInfo } from 'hooks';
 import { ExtensionLoginButton } from 'UI/extension/ExtensionLoginButton';
 import { LedgerLoginButton } from 'UI/ledger/LedgerLoginButton';
 import { WalletConnectLoginButton } from 'UI/walletConnect/WalletConnectLoginButton';
 import { WebWalletLoginButton } from 'UI/webWallet/WebWalletLoginButton';
+
 import styles from './unlockPageStyles.scss';
+import globalStyles from 'assets/sass/main.scss';
 
-export interface Props {
+export interface UnlockPagePropsType {
```

Comment:
what to do? breaking but Props was really a bad choice of name. I think still revert and redo naming on major version change

Reply:
Reverted and commented. 👍